### DISCUSSION TITLE
Assets public URL not HTTPS

### DISCUSSION THREAD

**Discussion by justbane** [pimcore/pimcore#10532] [Q&A / Support]
Using Pimcore 6.9.6

Assets Public URL and Deeplink urls are shown as "http" not as the protocol of the site (https)

This site is behind a LB - I understand that it is likely reading the insecure connection between the LB and the webserver... But we are passing in the standard X-Forwarded-Proto header as https

Any thoughts on how I can fix this? Config variable?

Thanks for any help

See screenshot

![Screen Shot 2021-10-07 at 10 01 18 AM](https://user-images.githubusercontent.com/4744199/136436397-9b47eb8a-7b97-42ec-87c8-5f1918a9c0f7.png)

**Comment by NiklasBr** [comment]
Make sure you are also using [setTrustedProxies()](https://symfony.com/doc/current/deployment/proxies.html) in your config, in 6.9.x I think you could add it to `web/app.php`.

**Reply by jdreesen** [reply]
Another place to put this config would be `app/startup.php` (see: https://github.com/pimcore/pimcore/blob/6.9/lib/Bootstrap.php#L144-L149) or configure it through the `TRUSTED_PROXIES` env variable (see the lines below). 

/edit: [here it is in the docs](https://pimcore.com/docs/pimcore/6.9/Development_Documentation/Getting_Started/Configuration.html#page_Adding-logic-to-the-startup-process)
